MetaCocktailsSwiftData/Model/Cocktails/Classics/WardEight.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  WardEight.swift
+//  MetaCocktailsSwiftData
+//
+//  Created by James Menkal on 8/17/24.
+//
+
+import Foundation
+
+
+var wardEight = Cocktail(cocktailName: "Ward Eight",
+                         glasswareType: .stemmedGlassware,
+                         garnish: [.halfOrangeWheel, .seasonalBerries],
+                         ice: nil,
+                         author: Author(person: "The Fairbanks Cocktail Popular",
+                                        place: "Boston Herald, sec. 2, 12",
```

Comment:
Same thing as the page number thing above, I think this should just be Boston Herald, with the year below.